### PR TITLE
comments view polish

### LINKED ISSUE #210504: indicate number of replies before comment in accessible view
So that screen reader users know what to expect, IE what is available to go to, we should indicate the number of replies before the comment, as it could be lengthy.

Screenshot 2024-04-16 at 1 12 42 PM

### LINKED ISSUE #210505: `$` in GHPR accessible view comment usernames
1. enable screen reader mode
2. focus a GHPR comment which has a few replies
3. Usernames have `$` 🐛 

Screenshot 2024-04-16 at 1 12 42 PM

### PR DESCRIPTION
**Pull request by meganrogge**
fix #210505
fix #210504

cc @alexr00

### CODE REVIEW

**Comment by meganrogge** [comment]
test comment

File: src/vs/workbench/contrib/comments/browser/commentsView.ts [hunk 1]
```diff
@@ -316,61 +316,70 @@ export class CommentsPanel extends FilterViewPane implements ICommentsView {
...
+					replyCount,
 					(typeof element.comment.body === 'string') ? element.comment.body : element.comment.body.value
 				) + replies;
 			} else {
 				return accessibleViewHint + nls.localize('resourceWithCommentLabelFile',
-					"{0} in {1}, comment: {2}",
+					"{0} in {1},{2}comment: {3}",
 					element.comment.userName,
 					basename(element.resource),
+					replyCount,
```
Comment (meganrogge) [inline src/vs/workbench/contrib/comments/browser/commentsView.ts:363]: test comment

Comment (meganrogge) [reply src/vs/workbench/contrib/comments/browser/commentsView.ts:363]: sdfsd

File: src/vs/workbench/contrib/comments/browser/commentsView.ts [hunk 2]
```diff
@@ -316,61 +316,70 @@ export class CommentsPanel extends FilterViewPane implements ICommentsView {
...
 					(typeof element.comment.body === 'string') ? element.comment.body : element.comment.body.value
 				) + replies;
 			}
 		}
 	}
 
-	private getRepliesAsString(node: CommentNode, type: 'ariaLabel' | 'accessibleViewContent'): string {
-		if (!node.replies.length || type === 'ariaLabel') {
+	private getRepliesAsString(node: CommentNode, forAriaLabel?: boolean): string {
```
Comment (meganrogge) [inline src/vs/workbench/contrib/comments/browser/commentsView.ts:370]: testing

**Review by amunger — APPROVED** [review]

**Pull request merged**